recipes/python-subunit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-subunit" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-subunit-{{ version }}.tar.gz
+  sha256: 042039928120fbf392e8c983d60f3d8ae1b88f90a9f8fd7188ddd9c26cad1e48
+
+build:
+  noarch: python
+  number: 0
+  script: "python -m pip install . -vv"
+  
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - extras
+    - testtools >=0.9.34
+    - six
```

Comment:
`six` doesn't appear to be used except for 1 unit test (which aren't needed here)?

Reply:
I verified it and `six` is needed to run `subunit`